test/verify/check-system-services

```diff
@@ -328,7 +328,10 @@ Unit=test.service
         self.toggle_onoff()
         self.check_service_details(["Failed to start", "Automatically starts"], ["Start", "Clear 'Failed to start'", "Disallow running (mask)"], True)
         if not user:
-            b.assert_pixels("#service-details", "details-test-fail", ignore=[".cockpit-log-panel .pf-c-card__body", ".pf-c-switch__toggle"])
+            b.assert_pixels("#service-details", "details-test-fail",
+                            # in medium layout we sometimes get a scrollbar depending on how may test-fail logs exist
```

Comment:
"many"

Reply:
Ugh -- I'll fix the typo in a follow-up commit in #16942, ok?